kaztron/cog/dice.py

```diff
@@ -84,6 +86,25 @@ def __init__(self, bot):
         await self.bot.say("[{}]\n**Sum:** {:d}".format(' '.join(rolls_str), total))
         logger.info("Rolled FATE dice: {!r} (sum={})".format(rolls_str, total))
 
+    # strip_list = map(str.strip, lines)
+    @commands.command(pass_context=True, ignore_extra=False, no_pm=False)
+    @in_channels(ch_allowed_list)
+    async def choose(self, ctx, *, choices: str):
+        "Need some help making a decision? Just give the .choose command two or more options(separated by ,) and it will choose for you!"
+        choices = list(map(str.strip, choices.split(",")))
```

Comment:
Add above this:

```
logger.info("choice: {}".format(message_log_str(ctx.message)))
```

(every command in KazTron logs when a command is triggered, using a similar info call)